Source/CombatExtended/CombatExtended/SuppressionUtility.cs

```diff
@@ -91,7 +92,7 @@ private static bool GetCoverPositionFrom(Pawn pawn, IntVec3 fromPosition, float
         {
             List<IntVec3> cellList = new List<IntVec3>(GenRadial.RadialCellsAround(pawn.Position, maxDist, true));
             IntVec3 bestPos = pawn.Position;
-            interceptors = pawn.Map.listerThings.ThingsInGroup(ThingRequestGroup.ProjectileInterceptor).Select(t => t.TryGetComp<CompProjectileInterceptor>()).ToList();
+            //interceptors = pawn.Map.listerThings.ThingsInGroup(ThingRequestGroup.ProjectileInterceptor).Select(t => t.TryGetComp<CompProjectileInterceptor>()).ToList();
```

Comment:
Git keeps history for us, if we really don't need this, it should go away instead of be left commented out.

Reply:
Removed it